flytekit/remote/remote.py

```diff
@@ -387,6 +387,14 @@ def get(
 
         logger.info(f"Nothing found from {flyte_uri}")
 
+    def fetch(self, uri: str, download_to: str, recursive: bool = True):
+        """
+        Much like `pyflyte fetch` this downloads the literals from a "flyte://..." flyte data URI.
```

Comment:
Can you add the full `:param uri:` comments here so it gets fulled documented? I suspect this will be used a bunch in notebooks.

Reply:
done